contracts/validators/CredentialAtomicQueryMTPValidator.sol

```diff
@@ -7,13 +7,9 @@ import "../interfaces/ICircuitValidator.sol";
 import "../interfaces/IVerifier.sol";
 import "../interfaces/IState.sol";
 
-contract CredentialAtomicQueryMTPValidator is
-    OwnableUpgradeable,
-    ICircuitValidator
-{
+contract CredentialAtomicQueryMTPValidator is OwnableUpgradeable, ICircuitValidator {
```

Comment:
I don't know whether it was discussed and decided to leave it as it is due to the tight schedule, but `CredentialAtomicQueryMTPValidator` and `CredentialAtomicQuerySIGValidator` should've been refactored to extract common functionality to some parent class.
Up to you how to solve it and whether to solve it at all...

Reply:
We can do it later.